### DISCUSSION TITLE
Check-off list items in discussions, issue descriptions, and comments with a single tap in view mode

### DISCUSSION THREAD

**Discussion by MaxM-FF** [community/community#15111] [Mobile] [answered]
Currently the only way to check-off a list item on native mobile is by entering into edit mode of either a discussion, an issue description, or a comment, and typing out the letter "x" within square brackets.

Please add the ability to directly check-off list items by way of a single tap.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback on this! The ability to check off a list item is coming to Android, soon (and already works on iOS).

**Reply by MaxM-FF** [reply]
That's so great that Android will also be receiving this capability, it'll be so much more convenient and efficient to work with! Thanks for the update and reply @brianlovin!

**Reply by Jalvarez30** [reply]
Okay, I'm new at all this period, but I always want it to learn. Is there anyone that can help me or point me in the right direction at least, something wanted to have a better understanding of. I enjoy help ours, and I believe that I can accomplish just that here and I would really appreciate if someone could lend a helping hand please.
